stories/features/shields/components/interfaceControls.tsx

```diff
@@ -3,46 +3,118 @@
  * You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 import * as React from 'react'
-import { Toggle, Stat, GridLabel } from '../../../../src/features/shields'
+
+// Feature-specific components
+import {
+  ShowMoreIcon,
+  Toggle,
+  Stat,
+  ResourcesSwitchLabel,
+  EmptyButton,
+  ToggleFlex,
+  ToggleGrid
+} from '../../../../src/features/shields'
+
+// Component groups
+import BlockedResources from './blockedReources/blockedResources'
+import StaticList from './blockedReources/staticList'
+
+// Fake data
 import locale from '../fakeLocale'
 import data from '../fakeData'
 
 interface Props {
   enabled: boolean
+  sitename: string
+  favicon: string
 }
 
-// Fake stuff
 interface State {
   blockAds: boolean
-  blockPopups: boolean
-  blockImages: boolean
+  connectionsEncrypted: boolean
+  openTotalAdsTrackersBlockedList: boolean
+  openConnectionsEncryptedList: boolean
 }
 
 export default class InterfaceControls extends React.PureComponent<Props, State> {
   constructor (props: Props) {
     super(props)
     this.state = {
       blockAds: false,
-      blockPopups: false,
-      blockImages: false
+      openTotalAdsTrackersBlockedList: false,
+      connectionsEncrypted: false,
+      openConnectionsEncryptedList: false
     }
   }
 
+  get totalAdsTrackersBlockedList () {
```

Comment:
using `get` accessors for non-simple properties, and without a corresponding `set` is an anti-pattern. This is a function which creates a new set of elements each time it is accessed, so should look like a function at the call site.

Reply:
❤️getters :(